device_doctor/lib/src/ios_device.dart

```diff
@@ -127,7 +127,46 @@ class IosDevice implements Device {
 
   @override
   Future<void> recover() async {
-    // Restarts the device first.
-    await eval('idevicediagnostics', <String>['restart']);
+    await restart_device();
+    await uninstall_applications();
```

Comment:
Should we uninstall first and then restart?

Reply:
Switched.